apiary.apib

```diff
@@ -229,23 +227,182 @@ You can authenticate in one of two ways when making API calls:
                 "message": "Access Token missing or invalid"
             }
         }
+
+## Regions [/regions]
+
+Lists all available regions.  Can be used in conjunction with `/brands` and `/marketplace` to provide an interactive gift card experience in your app or web offering.
+
+## list regions [GET /regions]
++ Request (application/json)
+    + Headers
+
+            Authorization: Bearer YOUR_ACCESS_TOKEN
+            
++ Response 200
+    + Attributes
+        + regions 
+            + id (number) - A numeric identifier for the region.
+            + name (string) - The name of the region.
+            + `image_url` (string) - An image for the region.
+        + info
+            + code (string) - Status code. [`INFO_MARKETPLACE_RETRIEVED_REGIONS`]
+    
+    + Body
+            
+            {
+                "regions": [
+                    {
+                        "id": 1,
+                        "name": "Canada",
+                        "image_url": "https://s3-us-west-2.amazonaws.com/staticimageskiind/flags/CA@3x.png"
+                    },
+                    {
+                        "id": 2,
+                        "name": "USA",
+                        "image_url": "https://s3-us-west-2.amazonaws.com/staticimageskiind/flags/US@3x.png"
+                    },
+                    {
+                        "id": 3,
+                        "name": "Global",
+                        "image_url": "https://s3-us-west-2.amazonaws.com/staticimageskiind/flags/GLBL@3x.png"
+                    },
+                    {
+                        "id": 4,
+                        "name": "Australia",
+                        "image_url": "https://s3-us-west-2.amazonaws.com/staticimageskiind/flags/AU@3x.png"
+                    }
+                ],
+                "info": {
+                    "code": "INFO_MARKETPLACE_RETRIEVED_REGIONS",
+                    "name": "Marketplace Regions Retrieved",
+                    "message": "A list of marketplace regions has been retrieved."
+                }
+            }
         
 
+## Brands [/brands]
+The `/brands` endpoint returns Giftbit's gift card brands available to you and facilitates building an elegant interactive gift card marketplace in your own app or web offering for your users.   It is recommended that you present brands via this endpoint passing in the appropriate restrictions for the user's selection (region, price range, etc), and then allow drilling down into the individual gift card offerings for that brand via a `/marketplace` call with the given `brand` as a parameter so the user can make their choice.  Note that this is how Giftbit builds its own interactive marketplace in our web application.
```

Comment:
`drilling down into the individual gift card offerings for that brand via a `/marketplace` call with the given `brand` as a parameter` 
they should also pass the region/currency and min/max price arguments used on `/brands`. since a vendor can have gifts in multiple currencies and different price ranges